src/main/java/controllers/TrainerController.java

```diff
@@ -54,4 +57,36 @@ public TrainerData getTrainer(String userIdentifier) throws UserDoesNotExistErro
 
         return trainer;
     }
+
+    /**
+     * Searches for all users with the given first name.
+     * @param firstName The first name of the member.
+     * @return A list of all the members with the given first name.
+     */
+    public List<MemberInfo> searchForMember(String firstName) {
+        Connection connection;
+        try {
+            connection = DriverManager.getConnection(databaseURL, databaseUsername, databasePassword);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        String query = "SELECT * FROM members NATURAL JOIN users WHERE first_name='" + firstName + "';";
+        List<MemberInfo> membersWithThatName = new ArrayList<>();
+        try {
+            Statement statement = connection.createStatement();
+            statement.executeQuery(query);
```

Comment:
in terms of writing queries and statements, using PreparedStatement might be simpler and would allow you to skip doing all the string concatenation when writing query. take a look at the member controller for a reference.
would be helpful later when we write complex queries.

Reply:
I can see there being a use case for PreparedStataments when you don't know where you getting the value for the query yet. However in most cases I feel that the read ability and traceability is better with the string concatenation. The variables are labeled so what was intended to go there and an IDE will take you to where it got the value. With the PreparedStatement you have to manually find where setString is called on the query then count question marks to see which one is being assigned.